### ISSUE TITLE
hozAlign:right for progress formatter

### ISSUE THREAD

**Issue by dbauszus-glx** [olifolkerd/tabulator#3829]
When we use hozAlign: right to make the progress bar go from right to left we get a wrong display.

The `.tabulator-cell > div > div` element should be set to right:0 to fix the display.

![image](https://user-images.githubusercontent.com/22201617/176482905-3021f0ec-944f-4a50-8df1-6525ff331473.png)

We are using this column setup for two progress formatter.

```JS
        {
          title: "Left",
          field: "left",
          hozAlign: "right",
          formatter: "progress",
          formatterParams: {
            color: ["#C0392B"],
            min: 0,
            max: 100
          }
        },
        {
          title: "Right",
          field: "right",
          formatter: "progress",
          formatterParams: {
            color: ["#16A085"],
            min: 0,
            max: 100
          }
        }
```

I created a codepen to show the issue.

https://codepen.io/dbauszus-glx/pen/rNdNXBo

**Comment by dbauszus-glx** [comment]
I think that hozAlign should be defined as formatter param.

I created a pull request for this. https://github.com/olifolkerd/tabulator/pull/3830

**Comment by dbauszus-glx** [comment]
I can confirm now that is fixed in 5.3 release.
